models/published/direct_care/C_LTCS/cltcs_measurements.sql

```
@@ -190,6 +190,56 @@ ldl_cholesterol_measurements as(
+        end as colour_mapping,
+        'eosinophil_count' as measurement_type
+    from {{ ref('int_eosinophil_count')}} eos 
+    inner join inclusion_list il on il.olids_id = eos.person_id
+    where eos.clinical_effective_date between dateadd(year, {{ measurement_cutoff }}, current_date()) and current_date()
+    qualify 
+        row_number() over (
+            partition by il.patient_id, eos.clinical_effective_date, eos.inferred_value
+            order by measurement_id
+        ) = 1
+),
+eosinophil_percentage_measurements as (
```

SQLFluff (4.2.2)

[error] Blank line expected but not found after CTE closing bracket.

(LT08)